crates/anstyle-svg/src/lib.rs

```diff
@@ -409,7 +418,14 @@ fn write_effects_in_use(buffer: &mut String, elements: &[adapter::Element]) {
     }
 }
 
-fn write_fg_span(buffer: &mut String, span: &str, element: &adapter::Element, fragment: &str) {
+fn add_classes(buffer: &mut String, classes: Vec<&str>) {
```

Comment:
As this is a helper for `write_fg_span`, it should go after it so the attention is drawn to the more important item